apps/allocations/app/components/Panel/NewAllocation.js

```diff
@@ -171,86 +181,90 @@ class NewAllocation extends React.Component {
-        input={
-          <DropDown
-            name="budget"
-            items={budgets.map(b => b.name)}
-            selected={budgets.indexOf(budgetValue)}
-            onChange={i => this.changeField({ target: {
-              name: 'budget',
-              value: budgets[i],
-            } })}
-            wide={true}
-          />
-        }
-      />
+      >
+        <DropDown
+          name="budget"
+          items={budgets.map(b => b.name)}
+          selected={budgets.indexOf(budgetValue)}
+          onChange={i => this.changeField({ target: {
+            name: 'budget',
+            value: budgets[i],
+          } })}
+          wide={true}
+        />
+      </Field>
     )
 
     const descriptionField = (
-      <FormField
+      <Field
         required
         label="Description"
-        input={
-          <DescriptionInput
-            name="description"
-            onChange={this.changeField}
-            value={descriptionValue}
-          />
-        }
-      />
+      >
+        <DescriptionInput
+          name="description"
+          onChange={this.changeField}
+          value={descriptionValue}
+        />
+      </Field>
     )
 
     const amountField = (
-      <FormField
+      <Field
         required
         label="Amount"
-        input={
+      >
+        <div css={`
+          display: flex;
+          flex-direction: column-reverse;
+          align-items: flex-end;
+          color: ${theme.textSecondary};
+          ${font({ size: 'small' })}
+        `}>
           <div css={`
             display: flex;
-            flex-direction: column-reverse;
-            align-items: flex-end;
-            color: ${theme.textSecondary};
-            ${font({ size: 'small' })}
+            line-height: 18px;
+            vertical-align: middle;
           `}>
-            <Text>
-              Available in Vault: {inVault} {tokenValue.symbol}
-            </Text>
-            <Text>
-              Available Budget: {remainingBudget} {tokenValue.symbol}
+            <Text css='margin-right: 6px'>
+              Available funds: {Number(inVault) > Number(remainingBudget) ? remainingBudget : inVault} {tokenValue.symbol}
             </Text>
-            <InputGroup css={`margin-bottom: ${GU}px; width: 100%`}>
-              <TextInput
-                name="amount"
-                type="number"
-                min={MIN_AMOUNT}
-                step="any"
-                value={amountValue}
-                onChange={this.changeField}
-                wide={true}
-                css={{ borderRadius: '4px 0px 0px 4px' }}
-              />
-              <CurrencyBox>{tokenValue.symbol}</CurrencyBox>
-            </InputGroup>
+            <Help hint="Available funds">
+              Available funds displays the amount of funds remaining within this allocation’s budget.
+              If this amount exceeds the organization’s vault funds, an accordingly adjusted value will be shown here.
```

Comment:
@javieralaves sorry I didn't leave feedback on this sooner! Now that I see it here, I want to propose two possible simplifications to this help text.

1. Only show the help icon when the available funds are less than the remaining amount in the budget, as that's _probably_ when a user will be confused. Then, the help text can say: "There's {remainingBudget} left in this budget until {periodEndDate}, but only {inVault} left in the organization's vault."
2. Always show the help icon, and when the vaults funds are the limiting factor, show the same phrase suggested above. When the vault funds aren't the limiting factor, just say the first part of the phrase, "There's {remainingBudget} left in this budget until {periodEndDate}."

What do you think?

Reply:
I spoke with @javieralaves in Keybase, and he thought Option 2 above sounded good

Let's change this help text, @PeterMPhillips